### PR TITLE
Added serverVersion to API; bumped API version to 2.2

### PR DESCRIPTION
**Pull request by maplesteve**
I'm extending https://github.com/gismo141/homebridge-server and would like to show the version of homebridge there by using the homebridge API.

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var hapLegacyTypes = require("hap-nodejs/accessories/types.js");
 var log = require("./logger")._system;
 var User = require("./user").User;
 var PlatformAccessory = require("./platformAccessory").PlatformAccessory;
+var serverVersion = require("./version");
```
Comment (KhaosT) [inline lib/api.js]: I'm sorry but where is this ``serverVersion`` thing?

Comment (maplesteve) [inline lib/api.js]: What do you mean by "is"? Where it's used? On line 27

Comment (KhaosT) [inline lib/api.js]: Nah never mind, I was thinking something else.

File: lib/api.js [hunk 2]
```diff
@@ -20,7 +21,10 @@ function API() {
   this._dynamicPlatforms = {}; // this._dynamicPlatforms[pluginName.platformName] = platform constructor
   
   // expose the homebridge API version
-  this.version = 2.1;
+  this.version = "2.1.1";
```
Comment (KhaosT) [inline lib/api.js]: Please bump this to number 2.2 instead of a string of "2.1.1"

**Comment by KhaosT** [comment]
Alright, looks good to me :)

**Comment by maplesteve** [comment]
Cool. Thanks!

**Comment by KhaosT** [comment]
I'll try to see if there are any other changes going to happen, if not, I'm going to push this to npm probably later today.

**Pull request merged**